### PR TITLE
Add new resources

### PR DESCRIPTION
**Pull request by harshhes**
### Updated Backend Section
- Added Django tutorial by Corey Schafer with link. **Corey Schafer** is one of the best Youtubers to teach python, I think this resource would be a great match also i have also learned Django through his videos only and I loved them.
- Added **MySQL** Database Resource with link

### CODE REVIEW

**Comment by harshhes** [comment]
Hey, How can i resolve this issue ?

File: README.md [hunk 1]
```diff
@@ -52,44 +52,57 @@ Add resources in other languages too :point_down:
...
-  * [Blockchain](#blockchain)
+- [:wave:  Code For Community welcomes you](#wave--code-for-community-welcomes-you)
+- [Gateway to Open Source | Best Programming Resources](#gateway-to-open-source--best-programming-resources)
+  - [Agenda:rocket:](#agendarocket)
+  - [Share your contribution](#share-your-contribution)
+    - [**Join Community :people_holding_hands: & Connect with me :handshake:**](#join-community-people_holding_hands--connect-with-me-handshake)
+  - [Contribution Guide :point_down:](#contribution-guide-point_down)
+  - [Other Languages](#other-languages)
+  - [Index](#index)
+  - [Android Development](#android-development)
```
Comment (Ashish-khanagwal) [inline README.md:63]: Remove from the line `56` to line `62`

File: README.md [hunk 2]
```diff
@@ -194,6 +208,13 @@ Add resources in other languages too :point_down:
 
 ### MongoDB
 - :file_folder: [MongoDB Documentation](https://www.mongodb.com/docs/manual/tutorial/getting-started/)
+
+### MySQL
+- :video_camera: [MySQL Tutorial By Mosh Hamedani](https://youtu.be/7S_tz1z_5bA) 
+
+### PostgreSQL
+- :video_camera: [PostgreSQL Tutorial By Corey Schafer](https://youtu.be/xaWlS9HtWYw)
+
```
Comment (Ashish-khanagwal) [inline README.md:217]: add MySQL and PostgreSQL in the index too under the backend section, and look at other modules in the index to add them properly.

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]
Please look at the requested changes and implement on them properly

**Comment by Ashish-khanagwal** [comment]
no changes are resolved, make sure to done with changes as requested

**Comment by harshhes** [comment]
shall i close this PR and create new instead of this as this is getting messed up. My current branch is 2 commits behind

**Comment by Ashish-khanagwal** [comment]
> shall i close this PR and create new instead of this as this is getting messed up. My current branch is 2 commits behind

don't close this PR, as it is not a good practice for contributing. navigate to your local repo and sync and fetch the upstream, commit the changes and make PR

**Comment by harshhes** [comment]
> > shall i close this PR and create new instead of this as this is getting messed up. My current branch is 2 commits behind
> 
> don't close this PR, as it is not a good practice for contributing. navigate to your local repo and sync and fetch the upstream, commit the changes and make PR

okay let me try and fix this

**Comment by Ashish-khanagwal** [comment]
Make sure to join Discord server, to ask further queries

**Review by Ashish-khanagwal — APPROVED** [review]
Looks good to me

Thank you for contributing

**Comment by harshhes** [comment]
> Looks good to me
> 
> Thank you for contributing
Happy to help!

**Pull request merged**